tools/make_release.py

```diff
@@ -10,9 +10,16 @@
 
```

Comment:
If we agree to use PEP8 as our style guide, then it must be possible to import this file as well without running the code. Create a main function and check if this was launched directly or imported. Code is available in PEP8 IIRC.